### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Ra1da35ma**
This looks a bit more appropriate

I might be wrong

### CODE REVIEW

File: content/index.md [hunk 1]
```diff
@@ -74,4 +74,4 @@ with through the years.
 
 This site attempts to collect all the related facts, rationale and techniques for Trunk-Based Development together
 in one place, complete with twenty-five diagrams to help explain things. All without using TBD as an acronym
-even ~~once~~ twice.
+even ~~once~~ or twice.
```
Comment (paul-hammant) [inline content/index.md]: Does the once word appear in strike-through form on your browser, or not?

Comment (Ra1da35ma) [inline content/index.md]: sure it does

I just feel it should still have an __or__ 

_Sent from my Huawei Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Comment (paul-hammant) [inline content/index.md]: I would agree if the once wasn't stricken through.  See here for examples - https://www.reddit.com/r/OutOfTheLoop/comments/1luith/what_do_people_mean_when_they_strike_through_some/

Comment (Ra1da35ma) [inline content/index.md]: thanks. That explains it all

😋😋

_Sent from my Huawei Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request closed without merging**